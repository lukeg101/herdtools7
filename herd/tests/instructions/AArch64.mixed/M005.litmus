AArch64 M005

(* Tests STR barrel shifters - UXTW *)

{ uint64_t x; 0:x0=1; 0:x2=x; int32_t 0:x3=0;}
P0                        ;
   STR W0, [X2, W3, UXTW];

exists (x=0x1)
