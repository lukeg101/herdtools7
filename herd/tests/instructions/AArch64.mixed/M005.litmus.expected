Test M005 Allowed
States 1
x=0x1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=0x1)
Observation M005 Always 1 0
Hash=2c1dc0ae671fb6f35401c54bde5b7845

